python/lsst/ip/diffim/makeKernelBasisList.py

```diff
@@ -232,6 +252,8 @@ def generateAlardLuptonBasisList(config, targetFwhmPix=None, referenceFwhmPix=No
         # http://iopscience.iop.org/0266-5611/26/8/085002  Equation 40
 
         # Use specializations for deconvolution
+        logger.debug("Target psf fwhm is the greater, deconvolution mode")
```

Comment:
I think this should be a higher level of logging output than `debug`. Maybe even `warn`, but at least `info`.

Reply:
Increased to `info`.